src/Compilers/CSharp/Test/Syntax/Parsing/CollectionLiteralParsingTests.cs

```diff
@@ -5365,8 +5364,8 @@ public void CastVersusIndexAmbiguity24_B()
     [Fact]
     public void CastVersusIndexAmbiguity24_C()
     {
-        // PROTOTYPE: This is not a great diagnostic.  Users could easily run into this and be confused.  Can we do
-        // better.  For example:
+        // This is not a great diagnostic.  Users could easily run into this and be confused.  Can we do
```

Comment:
Tracking issue?

Reply:
This particular case involves a dictionary element which is not currently supported. I've logged an issue to improve the diagnostic for the previous example `(A)[1]`: https://github.com/dotnet/roslyn/issues/68862.